README.md

```diff
@@ -128,10 +128,21 @@
 
 - Dino Run! - [Android](https://play.google.com/store/apps/details?id=io.github.epsi95.dinorun), - You are already familiar with this game, you can play it in Chrome when there is no internet. I just added a little color to it. Also you can shout to make the little cute Dino to jump, isn't that cool?. By [Probhakar Sarkar](https://github.com/epsi95)
 
+- Wormo [Android](https://play.google.com/store/apps/details?id=com.tbuonomo.wormo), [iOS](https://apps.apple.com/us/app/wormo-the-game/id1530797595) - Meet Wormo the gooey worm in this wonderful casual game ! Think you can put wormo in the nest? Think again, it requires an extraordinary technique. 
+By Tommy Buonomo ([Frenchie Games])
+
+#### Puzzle Games
+
+- Ripple Effect Puzzle - [Android](https://play.google.com/store/apps/details?id=com.tbuonomo.rippleeffectpuzzle), [iOS](https://apps.apple.com/us/app/id1521454937) - A sophisticated variation of Sudoku.
+By Tommy Buonomo ([Frenchie Games])
+
 ## Non-English Content
 
 - [Mandarin Chinese (中文)](https://github.com/flame-engine/awesome-flame/blob/main/CONTENT_ZH.md)
 
 ## Contributing
 
 [Please read the contribution guidelines](https://github.com/flame-engine/awesome-flame/blob/main/CONTRIBUTING.md)
+
+
+[frenchie games]: https://frenchiegames.app/#/
```

Comment:
This link should be on your entries, and not here, right @impulse ?

Reply:
Yes

Reply:
@tommybuonomo this link still has to be moved.